client/pages/section.js

```diff
@@ -81,7 +70,12 @@ class Section extends React.Component {
               data = { [data]: value };
             }
             const conditions = getConditions({ ...this.props.project, ...data }, CONDITIONS.project);
-            return this.props.update({ ...data, conditions });
+
+            this.props.update(data);
+
+            if (!this.props.isLegacy) {
+              this.props.update({ conditions })
```

Comment:
;

Reply:
do you remember that time when I added semicolons to lint, and then ran `lint --fix`, and then removed it, and then was going to add it back? Yeah that. I wont do it here, but I'll submit a separate PR to sweep all of them